.clang-format

```
@@ -0,0 +1,14 @@
+BasedOnStyle: "Google"
+
+AlignAfterOpenBracket:        Align
+AlignArrayOfStructures:       Left
+AlignConsecutiveAssignments:  AcrossComments
```

Can we restrain ourselves a bit on vertical alignment, at least this much?
```suggestion
AlignConsecutiveAssignments:  Consecutive
```